src/llfs/data_packer.test.cpp

```diff
@@ -189,4 +189,52 @@ TEST(DataPackerTest, ParallelDataCopyFalse)
   run_parallel_copy_test(false, 0);
 }
 
+void run_pack_record_test(const usize count)
+{
+  struct MyTemp {
+    int data;
+  };
+  const u64 mem_size = 64;
+  std::array<u8, mem_size> memory;
+  memory.fill(0);
+
+  std::array<u8, mem_size> expected;
+  expected.fill(0);
+
+  llfs::DataPacker packer{llfs::MutableBuffer{memory.data(), memory.size()}};
+
+  EXPECT_EQ(packer.space(), mem_size);
+
+  // allocate space for first lot based on 'count'
+  if (count != 1) {
+    [[maybe_unused]] auto p_buff = packer.template pack_record(batt::StaticType<MyTemp>{}, count);
```

Comment:
Please consider asserting something about the returned pointer instead of ignoring the value.

Reply:
sure